src/Option.tsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import OptionPage from './components/OptionPage';
+/*global chrome*/
```

Comment:
![image](https://user-images.githubusercontent.com/7676354/61025665-5e3a5000-a3ec-11e9-8c5c-798a7246218e.png)
It seems like this should be at the top of the file. Works only if it is moved to the top.

Reply:
? It works...